auth-api/tests/unit/api/test_org.py

```diff
@@ -584,10 +584,6 @@ def test_update_org(client, jwt, session, keycloak_mock):  # pylint:disable=unus
     # assert no contacts
     assert len(dictionary.get('contacts')) == 0
 
-    rv = client.put('/api/v1/orgs/{}'.format(org_id), data=json.dumps({'name': 'helo-duplicate'}),
-                    headers=headers, content_type='application/json')
```

Comment:
so what happens now when they try to update the name?
can we assert that condition as well?

Reply:
Yeah. With this change, an invalid exception would be raised if the input json consists of name property.